Podfile

```diff
@@ -8,6 +8,7 @@ platform :ios, '11.0'
 
 target 'StreemNow' do
   pod 'StreemKit', '~> 0.18.0'
+  pod 'AppAuth', '~> 1.4'
```

Comment:
Perhaps we could comment here that this is _only_ needed if the app is implementing authentication of non-anonymous users. (i.e. everything except for accepting an invite to start a streem as an anonymous user)

Reply:
As a side note, please don't refer to users logging in via an invitation as "anonymous users" in public documentation. They're not actually anonymous if they were invited via phone number or email address. We do associate all Streems with the same phone/email and company under a single user ID.